src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -243,25 +243,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           isLoading={isEventsLoading}
           {...scheduleNGWithAttendees}
         />
-      </div>
-      {!isEventsLoading && (
-        <div className="NavBarWrapper__download-buttons">
-          {isShowPersonalDownloadBtn && (
+        {!isEventsLoading && (
+          <div className="NavBarSchedule__downloadButtons">
+            {isShowPersonalDownloadBtn && (
+              <Button
+                onClick={downloadPersonalEventsCalendar}
+                customClass="NavBarSchedule__downloadScheduleBtn"
```

Comment:
```suggestion
                customClass="NavBarSchedule__download-schedule-button"
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23